packages/flutter_template_images/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 4.0.1
 
-* Updates package description.
+* Updated package description.
```

Comment:
Updates

(I got tired of the inconsistencies a while ago and added https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style)

Reply:
Ah nice!